README.md

```diff
@@ -737,6 +146,9 @@ You can use the following commands to run:
 * A specific suite of tests: ``rake test:unit``, ``rake test:integration``, ``rake test:integration:blob``, etc.
 * one particular test file: ``ruby -I"lib:test" "<path of the test file>"``
 
+To run ASM VCR tests:
```

Comment:
Set `INTEG_RECORDED` environment variable to `true` & then run rake `test:recorded`